pynicotine/gtkgui/application.py

```diff
@@ -344,8 +344,8 @@ def on_invalid_password(self):
         from pynicotine.gtkgui.widgets.dialogs import OptionDialog
 
         title = _("Invalid Password")
-        msg = _("User %s already exists, and the password you entered is invalid. Please choose another username "
-                "if this is your first time logging in.") % config.sections["server"]["login"]
+        msg = _("Either user %s already exists, or the password you entered is invalid."
+                " If this is the first time you log in, please choose a different username.") % config.sections["server"]["login"]
```

Comment:
"...time you log in" doesn't make sense.

Reply:
How would you put it?

Reply:
What was wrong with `your first time logging in`?

Reply:
Strictly speaking there isn't, but the proposed construction is how a native speaker would formulate it.

Reply:
An alternative way of putting it is `If this is the first time you have logged in`.